src/test/test-fileio.c

```diff
@@ -444,7 +444,9 @@ static void test_write_string_file_verify(void) {
         _cleanup_free_ char *buf = NULL, *buf2 = NULL;
         int r;
 
-        assert_se(read_one_line_file("/proc/version", &buf) >= 0);
+        r = read_one_line_file("/proc/version", &buf);
+        if (IN_SET(r, -EACCES, -EIO))
+                return;
```

Comment:
The check for other errors is now gone. It should still be there: `assert_se(r >= 0);` or so.